python/swiftnav/track.pyx

```diff
@@ -396,10 +404,10 @@ cdef class CN0Estimator:
 
   def __cinit__(self, **kwargs):
     cn0_est_init(&self._thisptr,
-                 self._thisptr.bw,
-                 self._thisptr.cn0_0,
-                 self._thisptr.cutoff_freq,
-                 self._thisptr.loop_freq)
+                 kwargs['bw'],
+                 kwargs['cn0_0'],
+                 kwargs['cutoff_freq'],
+                 kwargs['loop_freq'])
 
   def update(self, I, Q):
     """
```

Comment:
In general, I don't think the tracking stuff in libswiftnav python has tests, so adding some sanity tests would be great.